tests/regression/runner.py

```diff
@@ -689,6 +694,68 @@ def test_generate_address(self):
 
         eval_stat(time_cost, "generate_address")
 
+    def test_send_trytes(self):
+        logging.debug(
+            "\n================================send_trytes================================"
+        )
+        # cmd
+        #    0. single 2673 trytes legal transaction object
+        #    1. multiple 2673 ligal trytes transaction object
+        #    2. single 200 trytes illegal transaction object
+        #    3. single single 3000 trytes illegal transaction object
+        #    4. single unicode illegal transaction object
+        #    5. empty trytes list
+        #    6. empty not trytes list object
+        rand_trytes = []
+        for i in range(2):
```

Comment:
Why this should do multiple times?

Reply:
This section give me two trytes with different tips. The two trytes are treated as different trytes, even though their body are both all 9.